doc/basic_configuration.rst

```diff
@@ -55,17 +59,34 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+You need to have a `local smtp service configured <https://github.com/ros-infrastructure/buildfarm_deployment#setup-master-for-email-delivery>`_ for email notifications.
+Note that even when you remove these global email notification settings
+some jobs will still send notification emails to package specific email addresses.
+
 
 Update URLs to point to custom build farm
 -----------------------------------------
 
-Change the Jenkins URL in the entry point yaml to point to your earlier
-provisioned Jenkins master::
+Change the ``rosdistro_index_url`` in your ``ros_buildfarm_config``'s 
+``index.yaml`` to point to the ``rosdistro`` repository that defines the distro(s) that your buildfarm should build.
+
+**Important:** Note that this is the `rosdistro <https://github.com/ros/rosdistro>`_'s ``index.yaml``
+(which points to the files that define which packages are included in the distro and respective caches),
+and **not** the ``index.yaml`` in ``ros_buildfarm_config`` which you are editing.
```

Comment:
Isn't the previous sentence already clarifying which repository the `index.yaml` file belongs to? This seems redundant to me.

Reply:
Okay. I removed it in the last commit.
Personally I felt that since both files have the same name, stressing the difference here would be a good idea to avoid confusion.